__tests__/MoreScreen.test.tsx

```diff
@@ -1,25 +1,37 @@
 import React from 'react';
-import { render, fireEvent } from '@testing-library/react-native';
+import { render, fireEvent, waitFor } from '@testing-library/react-native';
 import { Provider } from 'react-redux';
 import configureStore from 'redux-mock-store';
 import MorePage from '../lib/screens/MorePage';
 import moxios from 'moxios';
 import { User } from '../lib/models/user_class';
 import { Linking } from 'react-native';
+import { useTheme } from '../lib/components/ThemeProvider';
 
-// Mock the ThemeProvider
+// Create a mock Redux store
+const mockStore = configureStore([]);
+const store = mockStore({
+  navigation: {
+    navState: 'more', // Add mock navigation state if needed
+  },
+  theme: {
+    darkMode: false, // Add mock theme state if needed
+  },
+});
+
+// Mock the ThemeProvider with spies
 jest.mock('../lib/components/ThemeProvider', () => ({
-  useTheme: () => ({
+  useTheme: jest.fn(() => ({
```

Comment:
Using jest.fn() to mock the function. Can make this a "variable" so it can be used across multiple tests